sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -82,6 +83,11 @@ export interface MessageHandlers<ReceivedMessageT> {
 
 export { MessagingError }
 
+// @public
+export interface OperationOptions extends TracingOptions {
```

Comment:
I made this change to mirror what we did in Event Hubs with the understanding that this is going to be one of the things we share between the 2 libraries and would eventually move to core-amqp

Also, doesn't `OperationOptions` also exist in `core-http`? So, if we were to re-use it from any of the core libraries, my bet would be on core-http rather than core-auth

Reply:
I guess my main point was more that it seems like we'd end up exposing 2 `OperationOptions`...one for the AMQP-based operations, and one for the HTTP-based operations (whether that's via core-auth or core-http doesn't matter for this argument). I wouldn't expect them to match given the extra pipeline options we have.

Since we've talked about supporting ATOM within this package before, I just wanted to bring it up to see if this would be a concern down the road (since service-bus is still in preview I don't see this as a blocker).

Reply:
Fair point, logged #8170 8172